### ISSUE TITLE
documentation wrong? intel-virtual-output works with external monitor on nvidia card

### ISSUE THREAD

**Issue by TheTesla** [Bumblebee-Project/Bumblebee#898]
https://github.com/Bumblebee-Project/Bumblebee/wiki/Multi-monitor-setup

> When the port is wired into the nvidia chip, you cannot currently expand the screen over monitors.

I have a Lenovo W520 laptop. I tried it out. I can expand the screen or I can clone it with intel-virtual-output and the DisplayPort seems to be wired to the nvidia card. The LVDS (internal screen) is on the intel card. I think this documentation is not correct?

**Comment by Lekensteyn** [comment]
Yes, documentation is outdated. Using nouveau (and maybe also intel-virtual-output or other techniques) you can in fact use multiple monitors.

**Comment by Stephen-Z** [comment]
> https://github.com/Bumblebee-Project/Bumblebee/wiki/Multi-monitor-setup
> 
> > When the port is wired into the nvidia chip, you cannot currently expand the screen over monitors.
> 
> I have a Lenovo W520 laptop. I tried it out. I can expand the screen or I can clone it with intel-virtual-output and the DisplayPort seems to be wired to the nvidia card. The LVDS (internal screen) is on the intel card. I think this documentation is not correct?

Have you got some bugs when using intel-virtual-output tool ? In my case, if I putting my external monitor on the right sides of my internal monitor , it just stop rendering ,every things stop responding except my mouse can still moving on it .
